### ISSUE TITLE
Set Password Url cannot open

### ISSUE THREAD

**Issue by bruce803** [overleaf/overleaf#674]
## Steps to Reproduce

1.In creating normal user with admin panel, the set passwd url cannot open
2.
3.

## Observed Behaviour

Hmmm...can’t reach this page
Try this

Make sure you’ve got the right web address: http://localhost
Search for "http://localhost" on Bing
Refresh the page
Details
Report this issue

Privacy statement

## Technical Info

* URL:
* Browser Name and version:Microsoft EDGE
* Operating System and version (desktop or mobile):

**Comment by mserranom** [comment]
Sorry, it's hard to follow what the problem is, would you mind providing the complete info, including what are the messages you see, logs, etc?

**Comment by DoctorNode** [comment]
I have had a similar problem. I'll try and be more precise than op.


## Steps to Reproduce



1. Install and start: Redis, MongoDB, and Overleaf
2. Create admin user following instructions on [wiki](https://github.com/overleaf/overleaf/wiki/Creating-and-managing-users)
3. Console generates set password link:
 http://localhost/user/password/set?passwordResetToken=691834a58d2e78451249746407d07e5a7c35d9468171765b384f3502f571f348
4. Modify to http://localhost:**PORT**/user/...."

## Expected Behaviour

Allow me to set password for account.

## Observed Behaviour

Redirects to localhost:PORT**/user/password/reset**



## Context

Attempting to set password using generated reset-token fails when attempting to navigate to link.
## Technical Info

Docker is running on top of Unraid.
## Attempted Troubleshooting

- Set _SHARELATEX_ALLOW_PUBLIC_ACCESS_ to **TRUE**
- No change.

**Comment by claystation** [comment]
What environment variables do you have defined? I had the same problem as you but i fixed it by only including the environment variables i needed. I removed all others that had something to do with the secure cookie or reverse proxy set up.
I do not use Docker compose as i run it on an Unraid instance and i run it on HTTP (which is fine by me as i do not have it publicly accessible).

The variables i have set are:

```
- SHARELATEX_APP_NAME
- SHARELATEX_NAV_TITLE
- SHARELATEX_SITE_URL
- SHARELATEX_ADMIN_EMAIL
- SHARELATEX_REDIS_HOST
- SHARELATEX_REDIS_PORT
- SHARELATEX_MONGO_URL
- SHARELATEX_RESTRICT_INVITES_TO_EXISTING_ACCOUNTS = true
- SHARELATEX_ALLOW_PUBLIC_ACCESS = false
- SHARELATEX_ALLOW_ANONYMOUS_READ_AND_WRITE_SHARING = false
- REDIS_HOST
- REDIS_PORT
- ENABLE_CONVERSIONS = true
- EMAIL_CONFIRMATION_DISABLED = true
```

Let me know if it helps, i fiunally got it to run!

**Comment by DoctorNode** [comment]
> What environment variables do you have defined? I had the same problem as you but i fixed it by only including the environment variables i needed. I removed all others that had something to do with the secure cookie or reverse proxy set up.
> I do not use Docker compose as i run it on an Unraid instance and i run it on HTTP (which is fine by me as i do not have it publicly accessible).
> 
> The variables i have set are:
> 
> ```
> - SHARELATEX_APP_NAME
> - SHARELATEX_NAV_TITLE
> - SHARELATEX_SITE_URL
> - SHARELATEX_ADMIN_EMAIL
> - SHARELATEX_REDIS_HOST
> - SHARELATEX_REDIS_PORT
> - SHARELATEX_MONGO_URL
> - SHARELATEX_RESTRICT_INVITES_TO_EXISTING_ACCOUNTS = true
> - SHARELATEX_ALLOW_PUBLIC_ACCESS = false
> - SHARELATEX_ALLOW_ANONYMOUS_READ_AND_WRITE_SHARING = false
> - REDIS_HOST
> - REDIS_PORT
> - ENABLE_CONVERSIONS = true
> - EMAIL_CONFIRMATION_DISABLED = true
> ```
> 
> Let me know if it helps, i finally got it to run!

Thank you so much! It works. I gave up for a long time and was trying again today. I removed all the unneeded environmental variables as suggested and it worked perfectly!
